examples/screenshot/main.py

```diff
@@ -0,0 +1,45 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import NumericProperty
+from kivy.properties import ObjectProperty
+from kivy.uix.boxlayout import BoxLayout
+
+
+Builder.load_string('''
+#:import screenshot plyer.screenshot
+<ScreenShotDemo>:
+    screenshot: screenshot
+    orientation: 'vertical'
+    padding: '50dp'
+    spacing: '20dp'
+    Label:
+        id: location_label
+        size_hint_y: None
+        height: sp(40)
+        text: 'Screenshot Location: ' + str(root.screenshot.file_path)
+
+    Button:
+        id: record_button
+        text: 'Take Shot'
```

Comment:
Screenshot, not "shot". "screenshot" is a real word and each big platform uses it in that shape:
* [Windows](https://support.microsoft.com/en-us/help/10933/windows-phone-take-a-screenshot)
* [Ubuntu](https://help.ubuntu.com/stable/ubuntu-help/screen-shot-record.html)
* [Fedora](https://fedoraproject.org/wiki/How_to_make_screenshots)